tox/constants.py

```diff
@@ -0,0 +1,56 @@
+"""All non private names (no leading underscore) here are part of the tox API.
+
+They live in the tox namespace and can be accessed as tox.[NAMESPACE.]NAME
+"""
+import sys as _sys
+
+
+class PYTHON:
+    CPYTHON_VERSION_TUPLES = [(2, 7), (3, 4), (3, 5), (3, 6), (3, 7)]
+    OTHER_PYTHON_INTERPRETERS = ['jython', 'pypy', 'pypy3']
+    _map = {'py': _sys.executable, 'py2': 'python2', 'py3': 'python3'}
+    _map.update({'py%s%s' % (major, minor): 'python%s.%s' % (major, minor)
+                 for major, minor in CPYTHON_VERSION_TUPLES})
+    _map.update({interpreter: interpreter for interpreter in OTHER_PYTHON_INTERPRETERS})
+    DEFAULT_FACTORS = _map
+    CURRENT_RELEASE_ENV = 'py36'
+    """Should hold currently released py -> for easy updating"""
```

Comment:
I think sphinx prefers comments instead of docstrings for variables; if it's a section marker then use an empty line to signal it better

Reply:
Really that surprises me. I though it was the other way round. Must have a look ...

Reply:
oh - that was for epydoc - there was a PEP going in that direction, but it was rejected: https://www.python.org/dev/peps/pep-0224/

I am not a very avid Sphinx user, so I have to up my game there I guess.